arches/app/models/resource.py

```diff
@@ -500,6 +500,26 @@ def get_documents_to_index(
         document["displayname"] = []
         document["displaydescription"] = []
         document["map_popup"] = []
+        try:
+            document["date_created"] = (
+                models.EditLog.objects.filter(
+                    resourceinstanceid=str(self.resourceinstanceid)
```

Comment:
```suggestion
                    resourceinstanceid=self.resourceinstanceid
```
Stringifying None probably would crash when attempting to cast this back to UUID.

Also, although strings are acceptable in a call like this, I'm concerned that it makes it "look safe" to use strings for UUID fields with the ORM, when in general we shouldn't, see #10877.